toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -461,7 +461,7 @@ chroot_and_install_rpms lua lua
 build_rpm_in_chroot_no_install lua-rpm-macros
 chroot_and_install_rpms lua-rpm-macros
 
-# Build tdnf-3.5.2
+# Build tdnf-3.5.6
```

Comment:
this comment can be removed.. it seems to be in the wrong place, and there's no reason to keep track of the version here

Reply:
Comment removed